.github/workflows/pr.yml

```diff
@@ -1,30 +1,51 @@
 name: PR Checks
+
 on:
   pull_request:
   pull_request_review:
-    types: [ submitted ]
+    types: [submitted]
 
 jobs:
   check_branch_name:
```

Comment:
I think I want to get rid of this check. It's artificial and generates noise. I guess we don't care about source branch name in the end.

Reply:
Meaning there is no need to check the branch name of the PR, right?